src/lib.rs

```diff
@@ -560,72 +560,67 @@ impl StacksDevnetApiK8sManager {
         namespace: &str,
     ) -> Result<StacksV2InfoResponse, DevNetError> {
         let client = HttpClient::new();
+        
+        // Log the constructed service URL and port
         let url = get_service_url(namespace, StacksDevnetService::StacksBlockchain);
-        let port =
-            get_service_port(StacksDevnetService::StacksBlockchain, ServicePort::RPC).unwrap();
+        let port = get_service_port(StacksDevnetService::StacksBlockchain, ServicePort::RPC).unwrap();
         let url = format!("http://{}:{}/v2/info", url, port);
-
+    
```

Comment:
Looks like your formatter isn't properly configured. (this and some of changes below).
We should add a check in CI

I'll follow up by DM